naughty/debian-testing/2463-no-pod-events-2

```diff
@@ -0,0 +1 @@
+b.click("#table-system_pod .create-container-in-pod")
```

Comment:
This is too thin. Can you please add some context like the affected test name,`Traceback:` and the actual exception, so that this doesn't catch unrelated errors?

Reply:
better?